daemon/info_unix.go

```diff
@@ -172,3 +211,30 @@ func parseInitVersion(v string) (types.Commit, error) {
 	}
 	return version, nil
 }
+
+// parseRuncVersion parses the output of `runc --version` and extracts the
+// "version" and "git commit" from the output.
+//
+// Output example from `runc --version`:
+//
+//   runc version 1.0.0-rc5+dev
+//   commit: 69663f0bd4b60df09991c08812a60108003fa340
+//   spec: 1.0.0
+func parseRuncVersion(v string) (version string, commit string, err error) {
+	lines := strings.Split(strings.TrimSpace(string(v)), "\n")
+	if len(lines) < 3 {
+		return "", "", errors.Errorf("unknown output format: %s", v)
+	}
+
+	parts := strings.Split(lines[0], " ")
+	if len(parts) == 3 {
+		version = strings.TrimSpace(parts[2])
+	}
+
+	parts = strings.Split(lines[1], ": ")
```

Comment:
maybe we should try to find a line that starts with "commit: "?

Reply:
I mean checking `for _, line := range lines { if strings.HasPrefix(line, "commit:") { ... }}` explicitly

Reply:
Yes, perhaps that's better; I just took the existing logic/code here, but can work on improving that